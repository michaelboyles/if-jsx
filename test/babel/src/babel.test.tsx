import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { If } from '../../../dist';

it('Positive if', () => {
    const tree = renderer
        .create(<><If condition={true}>Condition is true</If></>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
