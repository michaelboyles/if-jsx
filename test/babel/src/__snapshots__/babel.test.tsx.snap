// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Positive if 1`] = `"Condition is true"`;
